go/client/commands_non_osx.go

```diff
@@ -17,6 +17,10 @@ func getPlatformSpecificCommands(cl *libcmdline.CommandLine, g *libkb.GlobalCont
 	return []cli.Command{}
 }
 
+func StopLaunchdService(g *libkb.GlobalContext, label string, wait bool) error {
+	return fmt.Errorf("Unsupported on this platform")
```

Comment:
Yeah I could add a `Restart` method for osx and non_osx, but this seemed safer... e.g. less moving of code and refactoring...

I guess I'm stuck between doing the right way, or doing the way that affects less code